runtime-testsuite/pom.xml

```diff
@@ -119,6 +119,8 @@
                     <argument>${basedir}/test</argument>
                     <argument>-templates</argument>
                     <argument>${basedir}/resources/org/antlr/v4/test/runtime/templates</argument>
+                    <argument>-target</argument> <!-- TODO remove this before merging with upstream! -->
```

Comment:
I've created a wiki page. https://github.com/pboyer/antlr4/wiki/Merge-with-upstream

Maybe a ticket is a better place for that.